docs/big-data-cluster/deployment-guidance.md

```diff
@@ -64,6 +64,11 @@ kubectl config view
 
 After you have configured your Kubernetes cluster, you can proceed with the deployment of a new SQL Server big data cluster. If you are upgrading from a previous release, please see [How to upgrade [!INCLUDE[big-data-clusters-2019](../includes/ssbigdataclusters-ss-nover.md)]](deployment-upgrade.md).
 
+## Ensure you have storage configured
+It's important for most big data cluster deployments to have persistent storage. At this time, you need to make sure you have a plan for how you're going to provide persistent storage on the Kubernetes cluster *before* you deploy the BDC tools. 
+
+If you're using AKS and you plan to use dynamically provisioned storage, you should set up a persistent volume claim on your preferred storage class now. If you aren't planning to use AKS's dynamically provisioned storage (e.g. because you're deploying on your own kubeadm cluster), you'll need to ensure you have sufficient storage for a cluster of your desired scale available and configured for use. If you wish to customize how your storage is used, you should do this before proceeding. See [Data persistence with SQL Server big data cluster on Kubernetes](concept-data-persistence.md).
```

Comment:
This is a bit confusing. It should simply say:  "If you deploy in AKS, no storage setup is necessary as AKS has built \-in storage classes with dynamic provisioning enabled. You can customize the storage class (`default` or `managed-premium`) in the deployment configuration file. The built-in profiles use a `default` storage class. If you are deploying on a Kubernetes cluster you deployed using `kubeadm`, you'll need to ensure you have sufficient storage for a cluster of your desired scale available and configured for use. If you wish to customize how your storage is used, you should do this before proceeding. See [Data persistence with SQL Server big data cluster on Kubernetes](concept-data-persistence.md).

Reply:
It's possible that a user could be deploy on AKS and prefer some other storage class to the built-in ones. That's a possibility discussed in the linked doc. So I'd say "If you are deploying on a Kubernetes cluster you deployed using `kubeadm` or if you'd prefer to set local storage up manually..."